### PR TITLE
Desktop notifications support meteor 16

### PR DESCRIPTION
**Pull request by mjovkovic**

### CODE REVIEW

File: lib/server/push.api.js [hunk 1]
```diff
@@ -93,11 +93,20 @@ Push.Configure = function (options) {
...
+  const vets = GlobalVets.find({
+    $or: [{disabled: {$exists: false}}, {disabled: false}],
+    useFirebaseForMessaging: true
+  }).fetch();
   vets.forEach((vet) => {
     Meteor.call('raix:configure-push-connections', vet._id);
   });
 
+  if (desktopApps && desktopApps.length) {
+    desktopApps.forEach(appConfig => {
```
Comment (mslobodan) [inline lib/server/push.api.js]: @mjovkovic Maybe it  is better to rename `desktopApps` to `desktopAppsFirebaseConfigurations`.

File: lib/server/push.api.js [hunk 2]
```diff
@@ -320,11 +332,14 @@ function serverSend(raixNotification) {
 }
 
 function querySend(query, raixNotification) {
-  console.log('Push: Sending notifications for query: ', query);
+  console.log('Push: Sending notifications for query: ', JSON.stringify(query));
 
   let firebasePromises = [];
 
-  let appTokens = AppTokenUtil.getAppTokensBasedOnQuery(query);
+  let desktopAppId = raixNotification && raixNotification.payload && raixNotification.payload.isDesktop && raixNotification.payload.vetId || undefined;
```
Comment (mslobodan) [inline lib/server/push.api.js]: @mjovkovic It makes more sense to keep `isDesktop` flag directly in `raixNotification` instead in `raixNotification.payload`.

**Review by mslobodan — CHANGES_REQUESTED** [review]

File: lib/server/push.api.js [hunk 3]
```diff
@@ -320,11 +332,14 @@ function serverSend(raixNotification) {
 }
 
 function querySend(query, raixNotification) {
-  console.log('Push: Sending notifications for query: ', query);
+  console.log('Push: Sending notifications for query: ', JSON.stringify(query));
 
   let firebasePromises = [];
 
-  let appTokens = AppTokenUtil.getAppTokensBasedOnQuery(query);
+  let desktopAppId = raixNotification && raixNotification.isDesktop && raixNotification.desktopAppId || undefined;
```
Comment (mslobodan) [inline lib/server/push.api.js:339]: @mjovkovic You need to add validation for `isDesktop` and for `desktopAppId` to `packages/push/lib/common/notifications.js` in `_validateDocument` function.

**Review by mslobodan — CHANGES_REQUESTED** [review]

File: lib/server/push.api.js (hunk 3 above)
Comment (mslobodan) [reply lib/server/push.api.js:339]: @mjovkovic Add validation.

Comment (mjovkovic) [reply lib/server/push.api.js:339]: 👍

**Review by mslobodan — APPROVED** [review]

**Pull request merged**